cisco_umbrella_investigate/plugin.spec.yaml

```diff
@@ -931,3 +1083,70 @@ actions:
         description: Samples
         type: array
         required: true
+  timeline:
+    title: Timeline
+    description: Shows when a domain, IP or URL was given attribution of a particular security categorization or threat type (indicators of compromise)
+    input:
+      name:
+        title: Name
+        type: string
+        description: Domain name, IP address or URL
+        required: true
+        example: example.com
+    output:
+      timeline:
+        title: Timeline
+        description: Provided data for a queried domain name, IP address or URL
+        type: '[]timeline'
+        required: true
+  passive_dns:
+    title: Passive DNS
+    description: Provides historical data from our resolvers for domains, IPs, and other resource records
+    input:
+      resource_records:
+        title: Resource Records
+        type: string
+        description: Resource records for which historical data is provided
+        enum:
+          - Domain
+          - Name
+          - IP Address
+          - Raw Text
+          - Timeline
+        required: true
+        default: Domain
+        example: Domain
+      name:
+        title: Name
+        type: string
+        description: Domain name, IP address or text
+        required: true
+        example: example.com
+      recordType:
+        title: Record Type
+        description: The record types (A, NS, MX, TXT, CNAME) to return, use commas to separate multiple record types
```

Comment:
Can we allow these as both upper and lower case? e.g. `A,txt,CNAME,mx`

Reply:
API supported both upper and lower case, we test it and it's works. 
We can change description if you want :)